CodeEdit/Features/Tabs/Models/TabManager.swift

```diff
@@ -14,6 +14,8 @@ class TabManager: ObservableObject {
     /// Collection of all the tabgroups.
     @Published var tabGroups: TabGroup
 
+    @Published var focusActive: Bool
```

Comment:
This boolean is meant to represent whether or not the active editor (tab group - as it is known in the code) is being focused (filling the entire area).

Words are hard. 😅 

Xcode calls the editor containing the cursor the editor in focus. You can “Move focus” with command J. They also call expanding an editor “Focus this editor”. 🤷🏻‍♂️

Reply:
Ah okay. Yeah then I'd change it to 'focusIsActive' or 'editorFocusIsActive'

Reply:
Is it a good idea to use the "focus" term? This overlaps with other "focus" semantics which are used throughout the app. I think something like "singleEditorModeIsActive" might be less confusing

Reply:
That is also a good one

Reply:
Again, the variable now is focusActive - as in _focus_ the _active_ editor. Maybe `isFocusingActiveTabGroup` and we can later call it `isFocusingActiveEditor` when we rename tab group to editor.

Reply:
To further complicate things, "active" in this case, as in "the active editor" is used to indicate what is typically used as "focus" - where the cursor is at currently, the thing that was last clicked or tabbed into. So if we were to be accurate, it would then be named `isFocussingFocusedEditor`. 🤪

This focuses the editor (fills the space).
<img width="584" alt="image" src="https://github.com/CodeEditApp/CodeEdit/assets/806104/a0a00ce1-bc10-4b59-9104-48adcaa8429d">

This moves the focus to the editor (as in it moves the cursor in that editor)
<img width="427" alt="image" src="https://github.com/CodeEditApp/CodeEdit/assets/806104/2180351d-15b6-472d-b66b-a69f6cddc97f">

Reply:
Done! It is now `isFocusingActiveTabGroup`.